types/types.go

```diff
@@ -466,6 +466,16 @@ type ImageInspectInfo struct {
 	Os            string
 	Layers        []string
 	Env           []string
+	LayersData    []ImageInspectLayer
```

Comment:
Please move this one line up, to be close to `Layers` (and consistent with the `Inspect` implementations).

Or alternatively (less preferable?) update the `Inspect` implementations to match the order of fields here.